dependencies.go

```diff
@@ -0,0 +1,17 @@
+package vangoh
+
+import (
+	"time"
+)
+
+type Clock struct {
+	Now func() time.Time
+}
+
+var clock = Clock{}
+
+// Set up default implementations of all dependencies.
+// Tests can override these implementations as necessary.
```

Comment:
This seems to only serve a purpose in testing, as the only time it's run outside of test code is when calling `present := clock.Now()`.  If this is the case, can we make the Clock struct un-exported?

Also, the name "dependencies" for this file is confusing